.npmrc

```diff
@@ -1 +1,3 @@
-engine-strict=true
\ No newline at end of file
+engine-strict=true
+shell-emulator=true
+enable-pre-post-scripts=true
```

Comment:
We may want to remove this flag in the future, but I've enabled this flag for now to reduce the number of unnecessary changes.